src/acuvity/apexextend.py

```diff
@@ -35,7 +55,14 @@ def list_analyzer_names(self, group: Optional[str] = None) -> List[str]:
         """
         if self._available_analyzers is None:
             self._available_analyzers = self.list_analyzers()
-        return sorted([ a.id for a in self._available_analyzers if (group is None or a.group == group) and a.id is not None ])
+        analyzer_ids = [a.id for a in self._available_analyzers
```

Comment:
based on this, I remebered we need to add the list of patterns that can be added to the pattern detector, basically for each detector, list all possible matches available in our solution. 

This could be a documented also, but add a API too.

Reply:
no, it can be documented that there is a function which lists them ;)